app/assets/javascripts/components/feed_view.coffee

```diff
@@ -18,10 +18,10 @@
       className: 'feed'
       CSSTransitionGroup
```

Comment:
De hele transitiongroup is niet meer nodig omdat je de styling op de loading indicator hebt toegevoegd.